pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1795,4 +1795,10 @@ type ScrapeClass struct {
 	// TLSConfig section for scrapes.
 	// +optional
 	TLSConfig *TLSConfig `json:"tlsConfig,omitempty"`
+
+	// ExtraRelabelings allow setting extra default relabelings to all
+	// scrape objects.
+	//
+	// +optional
+	ExtraRelabelings []*RelabelConfig `json:"extraRelabelings,omitempty"`
```

Comment:
```suggestion
	Relabelings []*RelabelConfig `json:"relabelings,omitempty"`
```

I'm still not sure about my suggestion, but I think "Extra" might be redundant with the concept of scrape classes. What do you think?

Reply:
That's fine with me :)